modules/cnv-accessing-vmi-ssh.adoc

```diff
@@ -14,6 +14,8 @@ the `fedora-vm-ssh` service that forwards port 22 of the `<fedora-vm>` virtual
 machine to a port on the node:
 
 .Prerequisites
+* The virtual machine instance you want to access must be connected to the pod network.
```

Comment:
@SchSeba What do you think about: "* The virtual machine instance you want to access must be connected to the Pod network by using the `masquerade` binding method."

Reply:
I think that we customarily call this network Default network (but Pod network works fine for me).
I'm happy to add `masquerade` to the sentence.

Reply:
@dankenigsberg It occurs to me now that in most other locations we say "default Pod network."

Reply:
Hey @schseba, just wanted to follow up on this in case it wasn't clear. I think you made one change already (thanks!), but there are a couple other things that remain unaddressed. 

I think one of these suggestions might work - let me know your thoughts:

* The virtual machine instance you want to access must be connected to the default Pod network by using the `masquerade` binding method.

or

* The virtual machine instance you want to access must be connected to the default Pod network. Connections to the Pod network must use the `masquerade` binding method.

Note the capitalization of Pod and the backticks around masquerade. The rest is just slight rewording to match our style guide(s).

Reply:
Thanks for the help!

can you please take another look?